import-export-cli/cmd/set.go

```diff
@@ -115,13 +116,18 @@ func init() {
 		defaultExportDirectory = mainConfig.Config.ExportDirectory
 	}
 
+	if mainConfig.Config.TokenType != "" {
+		defaultTokenType = mainConfig.Config.TokenType
+	}
+
+
 	SetCmd.Flags().IntVar(&flagHttpRequestTimeout, "http-request-timeout", defaultHttpRequestTimeout,
 		"Timeout for HTTP Client")
 	SetCmd.Flags().StringVar(&flagExportDirectory, "export-directory", defaultExportDirectory,
 		"Path to directory where APIs should be saved")
-	SetCmd.Flags().StringVarP(&flagTokenType, "token-type", "t", "",
+	SetCmd.Flags().StringVarP(&flagTokenType, "token-type", "t", defaultTokenType,
 		"Type of the token to be generated")
-	SetCmd.Flags().StringVarP(&flagKubernetesMode, "mode", "m", "", "If mode is set to \"k8s\", apictl " +
+	SetCmd.Flags().StringVarP(&flagKubernetesMode, "mode", "m", utils.DefaultEnvironmentName, "If mode is set to \"k8s\", apictl " +
 		"is capable of executing Kubectl commands. For example \"apictl get pods\" -> \"kubectl get pods\". To go back " +
-		"to the default mode, set the mode to \"default\"")
+		"to the default mode, set the mode to \"default\" mode")
```

Comment:
Here mode word at the end is not required. It meant to set the mode to the value "default".